File: c/1066.c
#include <stdio.h>

int main(){
    int e, i, par=0, imp=0, pos=0, neg=0;
    for (i=0;i<5;i++){
        scanf("%d", &e);
        if (e%2 == 0){
            par++;
        }else {
            imp++;
        }
        if(e > 0){
            pos++;
        }else if (e < 0){
            neg++;
        }
    }
    printf("%d valor(es) par(es)\n", par);
    printf("%d valor(es) impar(es)\n", imp);
    printf("%d valor(es) positivo(s)\n", pos);
    printf("%d valor(es) negativo(s)\n", neg);
}

File: python/1066.py
par, imp, pos, neg = 0,0,0,0
for i in range(0, 5):
    e = int(input())

    if (e%2 == 0):
        par+=1
    else:
        imp+=1
    
    if(e > 0):
        pos+=1
    elif (e < 0):
        neg+=1
        

print("{:d} valor(es) par(es)".format(par));
print("{:d} valor(es) impar(es)".format(imp));
print("{:d} valor(es) positivo(s)".format(pos));
print("{:d} valor(es) negativo(s)".format(neg));